src/main.cpp

```diff
@@ -62,6 +62,10 @@ int main(int argc, char *argv[])
     Lith::instance()->windowHelperGet()->init();
 
     QQmlApplicationEngine engine;
+    engine.addImportPath("qrc:///");
+    QQuickStyle::setStyle("LithStyle");
+    //QQuickStyle::setFallbackStyle("Material");
```

Comment:
If I set a fallbackStyle it just uses that instead of my setStyle, so I commented out for now. :|